OpenSim/Simulation/Manager/Manager.h

```diff
@@ -95,38 +95,29 @@ class OSIMSIMULATION_API Manager
     double _ti;
     /** Final time of the simulation. */
     double _tf;
-    /** First dt in an integration. */
-    double _firstDT;
     
     /** Storage for the states. */
     std::unique_ptr<Storage> _stateStore;
 
-   int _steps;
-   /** Number of integration step tries. */
-   int _trys;
-   /** Maximum number of steps in an integration. */
-   int _maxSteps;
-   /** Flag for signaling a desired halt. */
-   bool _halt;
-   /** Minimum step size. */
-   double _dtMin;
```

Comment:
The properties of the `AbstractTool` are assigned to the `SimTK::Integrator` by `ForwardTool`, `CMCTool` and `RRATool` that is passed into the `Manager`. The private `Manager` members appear to be unused. In fact, where `_dtMax` is being used to test the initial *dt* should be replaced, since `_dtMax` is always `1.0` AFAICT.